test/living-html/htmlelement.js

```diff
@@ -0,0 +1,27 @@
+﻿"use strict";
+
+var jsdom = require("../..");
+
+exports["unknown elements should return HTMLUnknownElement"] = function (t) {
```

Comment:
Need tests that it works during parsing too.

Also need tests that elements without special classes, e.g. `<section>`, stay HTMLElement and not HTMLUnknownElement. Preferably an exhaustive test of such elements. And both parsing and explicit creation.